plotly_express/_doc.py

```diff
@@ -91,6 +91,11 @@
         "Ignored if `error_z` is `None`.",
     ],
     color=[colref, "Values from this column are used to assign color to marks."],
+    opacity=[
+        colref,
```

Comment:
in this case, opacity isn't a `colref` but a value between 0 and 1 actually, with a smart default.